src/main/java/org/openrewrite/kotlin/internal/KotlinTreeParserVisitor.java

```diff
@@ -68,7 +68,7 @@
 /**
  * PSI based parser
  */
-@SuppressWarnings("ConstantValue")
+@SuppressWarnings({"ConstantValue", "DataFlowIssue", "SameParameterValue", "unused"})
```

Comment:
Please remove DataFlowIssue suppression.
For future reference, it causes potential NPEs not to be shown in the IDE, so we shouldn't add it.